dams-tests/tests/common.rs

```diff
@@ -73,7 +70,7 @@ pub async fn setup(db: Database) -> ServerFuture {
     // Note: hard-coded to match the 2-service server with default port.
     let checks = vec![await_log(
         Party::Server,
-        TestLogs::ServerSpawned(SERVER_ADDRESS.to_string()),
+        TestLogs::ServerSpawned(SERVER_ADDRESS.to_string() + ":1113"),
```

Comment:
```suggestion
        TestLogs::ServerSpawned(format!("{}:1113", SERVER_ADDRESS)),
```
`+` works but using `format!` is the idiomatic way to do this.